Documentation/ReleaseProcess.md

```diff
@@ -0,0 +1,112 @@
+
+Using the command line, the process is pretty straightforward:
+
+```
+git checkout develop
+git pull
+git checkout -b release/x.y.z
+git push origin -u release/x.y.z
+```
+
+Make sure to update the field `s.version` in both of these files to the `x.y.z` number
+```
+WordPress-Aztec-iOS.podspec
+WordPress-Editor-iOS.podespec
+```
+
+Open a PR on github where you target the `master` branch with the `release/x.y.z` branch and call it `Release x.y.z`
+
+
+### Step 2: Testing the Integration ###
+
+Before going any further, It's normally good practice to test the Aztec integration into WordPress-iOS project, to make sure I won’t have to do the release process twice.
+
+Make sure WordPress-iOS’s podfile specifies the SAME (and latest) commit number for both of these:
+```
+pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => 'e0fc55abb4809b3b23b6d8b56791798af864025d'
+pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => 'e0fc55abb4809b3b23b6d8b56791798af864025d'
+```
+The things we want to look out for are:
+
+- That the pod can be properly installed in WPiOS.
+- That WPiOS builds properly.
+- That you can open a post through the App, using both the classic editor and Gutenberg.
+
+### Step 3: Merge the branch ###
+
+If the release PR is approved and all is working correctly in WP-iOS merge the release branch to master.
+
+### Step 4: Push the POD. ###
+
+*Automated Process*
+
+Create a new release on Github targetting the master branch and name it `Release x.y.z`
+
+Set a tag with the value `x.y.z` .  
+
+On the description field and the content of the changelog for this version
+
+Press the `Publish release` button
+
+At this moment the CI automation should notice your tag and after some minutes you should get the new pod publish on CocoaPods master trunk.
+
+If this for some reason fails chech the manual process bellow
+
+*Manual Process*
+
+Pushing the PODs is a bit different than in other repos due to the fact that this repo has two podspecs.
+
+The recommended steps are:
+
+Lint the core library
+```
+bundle exec pod lib lint WordPress-Aztec-iOS.podspec
+```
+
+If all is good proceed to
+```
+bundle exec pod trunk push WordPress-Aztec-iOS.podspec
+```
+
+Then lint the Editor library
+```
+bundle exec pod lib lint WordPress-Editor-iOS.podspec
+```
+
+If all is good, push it
+```
+bundle exec pod trunk push WordPress-Editor-iOS.podspec
+```
+
+### Step 5: Closing the milestone ###
+
+For simplicity Aztec uses a single milestone named “Next Stable”. The purpose of this milestone is to be used to assign to it all of the regular issues that are closed and merged into develop during the regular development process.
+
+At this point in the release process you should rename the milestone to the version you’re about to release.
```

Comment:
Nit: "you're about to release" might be better as "you just released" since we've already merged to `master` and pushed the POD at this point.